Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer 'python'
+      yamlFile '.jenkins/python.yaml'
+    }
+  }
+
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(
+      logRotator(
+        daysToKeepStr: '14',
+        numToKeepStr: '100',
+        artifactDaysToKeepStr: '30',
+      ))
+  }
+
+  stages {
+    stage('setup') {
+      steps {
+        sh 'apk add git gcc musl-dev linux-headers libffi-dev libressl-dev'
+        sh 'pip install --upgrade tox'
+      }
+    }
+
+    stage('test') {
+      steps {
+        sh 'tox'
```

Comment:
tox.ini missing(I cant see it)?

Reply:
already there: https://github.com/ClarkSource/aws-exporter/blob/master/tox.ini